docs/publishers/views.md

```diff
@@ -0,0 +1,116 @@
+# Views
+
+- [Introduction](#introduction)
+    - [Motivation](#motivation)
+    - [Concepts and Background](#concepts-and-background)
+- [Examples of using views](#examples-of-using-views)
+    - [Simple graph spec](#simple-graph-spec)
+    - [Vega graphs](#vega-graphs)
+    - [Maps](#maps)
+    - [Tables and Transforms](#tables-and-transforms)
+
+## Introduction
+
+### Motivation
+
+Producers and consumers of data [packages] want to have their data presented in tables and graphs -- "views" on the data.
+
+*Why? For a range of reasons -- from simple eyeballing to drawing out key insights.*
+
+<div class="mermaid">
+graph LR
+  data[Your Data] --> table[Table]
```

Comment:
Minor suggestion: we can use awesome fonts for table and graph
fa:fa-table, fa:fa-graph